linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/rule.yml

```diff
@@ -28,6 +28,7 @@ identifiers:
 
 references:
     disa: "1954"
+    nist: IA-2(4)
```

Comment:
Should this be IA-2(1), IA-2(2), IA-2(3), and IA-2(4)?

ref: https://nvd.nist.gov/800-53/Rev4/control/IA-2